tutorials/desktop/install-ubuntu-on-chromebook/install-ubuntu-on-chromebook.md

```diff
@@ -0,0 +1,109 @@
+
+To get to Developer Mode, we need to first reboot into Recovery Mode. On most Chromebooks, you do so by turning the device off, then holding down the `ESC` and `Refresh` keys while you press the Power button.
+
+Once in this mode, press `Ctrl-D`. You will be prompted with an opportunity to "turn OS verification OFF". Press Enter to do so.
+
+When you boot up your Chromebook, it will begin with a warning screen noting that "OS verification is OFF". You will need to press `Ctrl-D` to continue. Your device will now transition to Developer Mode.
+
+![]https://www.howtogeek.com/wp-content/uploads/2015/02/ximg_54f13e24a1e8f.jpg.pagespeed.gp+jp+jw+pj+ws+js+rj+rp+rw+ri+cp+md.ic.EsQ9doLxVp.jpg
+(Image credit: Chris Hoffman, How-to Geek)
+
+Every boot thereafter will also begin with that warning screen, and a need to press `Ctrl-D` to continue. Do not follow the onscreen instructions to turn OS verification on, or you risk wiping your machine's data and turning Developer Mode off.
+
+## Installing Ubuntu with crouton
+Duration: 4:00
+
+Empowered with Developer Mode, we will download the crouton script at this [link](https://goo.gl/fd3zc).
+
+Then we will open up a shell by pressing `Ctrl-Alt-T`, and then typing `shell`. Now we have a full bash shell at our fingertips.
+
+For a simple setup, we're going to type `sudo sh ~/Downloads/crouton -t unity`. This will download the Ubuntu 16.04 packages with the default Unity desktop environment.
+
+You'll see your terminal processing these packages one by one. This will take some time, so feel free to browse the web on ChromeOS, have a snack, or browse the web while having a snack.
+
+positive
+: **Alternative Installation Options**
+Depending on the specification of your device, the Unity desktop may be too demanding to work well for you. If you want a more lightweight system, you can install LXDE or XFCE by replacing `-t unity` with `-t lxde` or `-t xfce`. GNOME fans can also try `-t gnome` instead.
+You can also encrypt your chroot with a passphrase by adding `-e` to the end of the installation command before excuting it. This will require that you type in the encryption password you've chosen each time you enter the chroot environment.
+
+## Switching between OSes
+Duration: 2:00
+
+With the installation complete, you will be prompted to enter a username and a password. You will then be brought into a bare-bones Ubuntu setup. 
+
+Pressing `Ctrl-Alt-Shift-Back` and `Ctrl-Alt-Shift-Forward` will rotate you between ChromeOS and Ubuntu. Logging out of Ubuntu will drop you back into the ChromeOS terminal tab that ran the system.
+
+You can get back to Ubuntu in future sessions by typing `sudo startunity` into your bash shell.
+
+## Practical and fun uses for your system
+Duration: 3:00
+
+Like playing games? `sudo apt-get install steamv in a terminal gets you Valve's Steam gaming client for Linux-based desktop gaming on your Chromebook. If you're not a previous Steam user, you'll need to register an account.

```

Comment:
Please replace `sudo apt-get install` by `sudo apt install`.
And a typo: "steamv" -> "steam"